packages/salesforcedx-vscode-core/src/commands/forceOrgOpen.ts

```diff
@@ -6,24 +6,90 @@
  */
 
 import {
+  CliCommandExecutor,
   Command,
   SfdxCommandBuilder
 } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
 import { nls } from '../messages';
+import { telemetryService } from '../telemetry';
+import { getRootWorkspacePath } from '../util';
 import {
   EmptyParametersGatherer,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
+interface CLIOrgData {
+  status: number;
+  result: {
+    orgId: string;
+    url: string;
+    username: string;
+  };
+}
+
 class ForceOrgOpenExecutor extends SfdxCommandletExecutor<{}> {
   public build(data: {}): Command {
-    return new SfdxCommandBuilder()
+    const builder = new SfdxCommandBuilder()
       .withDescription(nls.localize('force_org_open_default_scratch_org_text'))
       .withArg('force:org:open')
-      .withLogName('force_org_open_default_scratch_org')
-      .build();
+      .withLogName('force_org_open_default_scratch_org');
+    if (process.env.SFDX_CONTAINER_MODE) {
+      builder.withArg('--urlonly').withJson();
+    }
+    return builder.build();
+  }
+
+  private buildUserMessageWith(orgData: CLIOrgData): string {
+    return nls.localize(
+      'force_org_open_container_mode_message_text',
+      orgData.result.orgId,
+      orgData.result.username,
+      orgData.result.url
+    );
+  }
+
+  private openBrowserWithCliResponse(cliResponseJSON: string | Buffer) {
+    try {
+      const cliOrgData: CLIOrgData = JSON.parse(cliResponseJSON.toString());
+      const authenticatedOrgUrl: string = cliOrgData.result.url;
+
+      channelService.appendLine(this.buildUserMessageWith(cliOrgData));
+
+      if (authenticatedOrgUrl) {
+        vscode.env.openExternal(vscode.Uri.parse(authenticatedOrgUrl));
+      }
+    } catch (e) {
+      telemetryService.sendErrorEvent(e.message, e.stack);
+    }
+  }
+
+  public execute(response: ContinueResponse<string>): void {
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new CliCommandExecutor(this.build(response.data), {
+      cwd: getRootWorkspacePath()
+    }).execute(cancellationToken);
+
+    if (process.env.SFDX_CONTAINER_MODE) {
+      channelService.showChannelOutput();
+      channelService.streamCommandStartStop(execution);
```

Comment:
This line will handle displaying the start and stop time for command execution and display the exit code.
`11:06:15.720 sfdx force:org:open --urlonly --json --loglevel fatal`
"User message"
`11:06:18.396 sfdx force:org:open --urlonly --json --loglevel fatal ended with exit code 0`